testdata/project-v3/PROJECT

```diff
@@ -5,23 +5,39 @@ repo: sigs.k8s.io/kubebuilder/testdata/project-v3
 resources:
 - api:
     crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: testproject.org
   group: crew
   kind: Captain
+  path: sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1
   version: v1
   webhooks:
+    defaulting: true
+    validation: true
     webhookVersion: v1
 - api:
     crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: testproject.org
   group: crew
   kind: FirstMate
+  path: sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1
   version: v1
   webhooks:
+    conversion: true
     webhookVersion: v1
 - api:
     crdVersion: v1
+  controller: true
+  domain: testproject.org
```

Comment:
we will not hide, not stored the command when it is == the projects domain?

Reply:
Its unlucky but an empty domain is valid so we can't hide it to mean that it is the default.

Reply:
I think we just need to add a small doc over what is and how works the PROJECT file in https://book.kubebuilder.io/reference/reference.html. However, in PV it is not a blocker for this one at all and be done as a follow up as well.